print.go

```diff
@@ -130,13 +139,6 @@ func Fprint(writer io.Writer, a ...interface{}) {
 	} else {
 		color.Print(Sprint(ret))
 	}
-
-	// Refresh all progressbars in case they were overwritten previously. Reference: #302
```

Comment:
Fixing concurrency issues when printing seemed to resolve this. See the test I added to validate this.